src/axom/mint/docs/sphinx/sections/fem.rst

```diff
@@ -0,0 +1,253 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+
+.. _sections/fem:
+
+Finite Elements
+-----------------
+
+The Finite Element Method (FEM) is a widely used numerical technique, employed
+for the solution of *partial differential equations* (PDEs), arising in
+*structural engineering analysis* and more broadly in the field of
+*continuum mechanics*.
+
+However, due to their generality and mathematically sound foundation,
+:ref:`sections/fem`, are often employed in the implementation of other numerical
+schemes and for various computational operators, e.g., interpolation,
+integration, etc.
+
+Mint provides basic support for :ref:`sections/fem` that consists:
+
+#. :ref:`LagrangeBasis` *shape functions* for commonly employed :ref:`CellTypes`
+#. Corresponding :ref:`Quadratures` (under development)
+#. Routines for forward/inverse :ref:`IsoparametricMapping`, and
+#. Infrastructure to facilitate adding *shape functions* for
+   new :ref:`CellTypes`, as well as, to :ref:`AddABasis`.
+
```

Comment:
Please mention alternative libraries, or acknowledge inspiration from other software and explain (or at least mention) why `mint::FiniteElement` was necessary.

Reply:
The 1st 2 paragraphs of this section motivate this. It is beyond the scope of this user guide to explain the theory of the Finite Element method. Especially, since Mint is not a Finite Element code. But, I can add some references.

Reply:
I feel it would help to elaborate the first two paragraphs somewhat further.  But if you're satisfied with the first two paragraphs as motivation for `mint::FiniteElement`, feel free to resolve this conversation.

Reply:
I am running low on bandwidth for this. I'll try to get to this in a later PR.